catt/cli.py

```diff
@@ -517,7 +517,13 @@ def get_alias_from_config(config, device):
 def get_device_from_settings(settings):
     device = settings.get("device")
     if device:
-        if not get_chromecast(device):
+        devices = get_chromecasts()
+        try:
+            if is_ipaddress(device):
+                next(d for d in devices if d.host == device)
+            else:
+                next(d for d in devices if d.name == device)
```

Comment:
Can you refactor these in a function called `first()` or similar? It'll be clearer what it does.

Reply:
How would I change the specific attribute I need to check in the if clause?

Reply:
Or did you just mean put this part of the logic in its own func?

Reply:
Sorry, I just noticed that the attribute was different. Never mind then.